source/write/delete.txt

```diff
@@ -0,0 +1,175 @@
+
+..    To start the lab, click the :guilabel:`Open Interactive Tutorial` button at the
+..    top of the page. To expand the lab to a full-screen format, click the
+..    full-screen button (:guilabel:`⛶`) in the top-right corner of the lab pane.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``sample_restaurants.restaurants`` collection
+from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the
+:ref:`<pymongo-get-started>` tutorial.
+
+Delete Operations
+-----------------
+
+You can perform delete operations in MongoDB by using the following methods:
+
+- ``delete_one()``, which deletes *the first document* that matches the search criteria
+- ``delete_many()``, which deletes *all documents* that match the search criteria
+
+Each delete method requires a **query filter** document, which specifies the
+search criteria that determines which documents to select for removal.
+For more information about query filters, see the 
+:manual:`Query Filter Documents section </core/document/#query-filter-documents>` in
+the MongoDB Server manual.
+
+Delete One Document
+~~~~~~~~~~~~~~~~~~~
+
+The following example uses the ``delete_one()`` method to remove a document in
+the ``restaurants`` collection with a ``name`` value of ``"Ready Penny Inn"``:
+
+.. literalinclude:: /includes/write/delete.py
+   :start-after: start-delete-one
+   :end-before: end-delete-one
+   :language: python
+   :copyable:
+
+Delete Multiple Documents
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example uses the ``delete_many()`` method to remove all documents
+in the ``restaurants`` collection with a ``borough`` value of ``"Brooklyn"``:
+
+.. literalinclude:: /includes/write/delete.py
+   :start-after: start-delete-many
+   :end-before: end-delete-many
+   :language: python
+   :copyable:
+
+Customize the Delete Operation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``delete_one()`` and ``delete_many()`` methods optionally accept additional
+parameters, which represent options you can use to configure the delete
+operation. If you don't specify any additional options, the driver does not customize
+the delete operation.
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Property
+     - Description
+
+   * - ``collation``
+     - | Specifies the kind of language collation to use when sorting
+         results. For more information, see :manual:`Collation </reference/collation/#std-label-collation>`
+         in the MongoDB Server manual.
+
+   * - ``hint``
+     - | Gets or sets the index used to scan for documents. 
+         For more information, see the :manual:`hint statement </reference/command/delete/#std-label-deletes-array-hint>`
+         in the MongoDB Server manual.
+
+   * - ``session``
+     - | An instance of ``ClientSession``.
+
+   * - ``let``
+     - | A Map of parameter names and values. Values must be constant or closed
```

Comment:
If Map is a data type, might be good to monospace

```suggestion
     - | A ``Map`` of parameter names and values. Values must be constant or closed
```

Reply:
Just a typo